rules.d/99-systemd.rules.in

```diff
@@ -9,6 +9,7 @@
 
 ACTION=="remove", GOTO="systemd_end"
 
+SUBSYSTEM=="tty", KERNEL=="ttyS*", DRIVERS=="serial8250", ATTR{port}=="0x0", ENV{SYSTEMD_READY}="0"
```

Comment:
I'd probably move it one line down. Doesn't matter at all, but I think it would make sense to define the general rule before the exception, and not the exception before the general rule.

Also, as usually, maybe addin a comment would be nice:

```
# Exclude 8250 serial ports with a zero IO port, as they are not usable until "setserial /dev/ttySxxx port …" is invoked.
```